canonical-web-and-design/ubuntu.com
# Build `/lxd` page

## Done

- Build `/lxd` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Copy doc](https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ/edit#), [Design](https://github.com/canonical-web-and-design/web-squad/issues/4892) 
- Demo at https://ubuntu-com-11378.demos.haus/lxd

## Issue / Card

Fixes [#4893](https://github.com/canonical-web-and-design/web-squad/issues/4893)

## Screenshots
<img width="1440" alt="update-lxd" src="https://user-images.githubusercontent.com/57550290/159470734-62d870bf-908d-4357-bf2e-5717df7c5eb6.png">


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
